k8s/environments/test/common-overlay/flux-patch.yaml

```diff
@@ -58,3 +58,52 @@ spec:
   values:
     java:
       image: sdshmctspublic.azurecr.io/pip/subscription-management:prod-e23c232-20220117145616
+
+
+
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: vh-user-api
+  namespace: vh
+  annotations:
+    fluxcd.io/automated: "true"
+    filter.fluxcd.io/java: glob:prod-*
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/vh/user-api:20210210.7
+    global:
```

Comment:
the global should be included by the environment helm release which should already be working, and if not this isn't the place to add it

Reply:
https://github.com/hmcts/vh-user-api/tree/feature/VIH-7155_AksSecretFileProvider/charts/vh-user-api
Not included in the repo the helmchart is pointing to.

Reply:
It's this file:
https://github.com/hmcts/sds-flux-config/blob/master/k8s/environments/test/test-helmrelease.yaml